docs/src/for-developers/how_turing_implements_abstractmcmc.md

```diff
@@ -140,20 +140,20 @@ end
 
 When doing Importance Sampling, we care not only about the values of the samples but also their weights. We will see below that the weight of each sample is also added to `spl.state.vi`. Moreover, the average 
 
-$$
+```math
 \frac 1 N \sum_{j=1}^N w_i = \frac 1 N \sum_{j=1}^N p(x,y \mid s_i, m_i)
-$$ 
+``` 
 
 of the sample weights is a particularly important quantity:
 
 * it is used to **normalize** the **empirical approximation** of the posterior distribution
-* its logarithm is the importance sampling **estimate** of the **log evidence** $\log p(x, y)$
+* its logarithm is the importance sampling **estimate** of the **log evidence** ``$\log p(x, y)$``
```

Comment:
Is this actually correct? That's two math delimiters it seems, both ``` `` ``` and `$`?

Reply:
I had to add the additional `$` to make it work when building the website locally (using `Turing.jl/docs/make.jl`). Here are the differences I observe:

This is how it looks locally when writing ` ``$\log p(x, y)$`` `:
![image](https://user-images.githubusercontent.com/55992574/180829941-dcb63dbe-7dba-44c1-886c-88a5a761bc1c.png)

However, here is what it looks like live:
![image](https://user-images.githubusercontent.com/55992574/180830121-ebacd740-58bf-4281-8dac-999798a86a40.png)

So I am assuming that the live build works differently than the local build, maybe because it is not build from the same `make.jl` command? But that's just a guess. 

If someone can point me to the "right" way to build the website locally, I'll give this another try.

Reply:
> If someone can point me to the "right" way to build the website locally, I'll give this another try.

I haven't tried myself yet unfortunately, but reproducing the build locally would involve running the build script in the (locally checked out) turing.ml repo, I assume. I guess you could use a local version of Turing.jl (with your changes) by replacing some hardcoded links to the Turing.jl repo or dev'ing Turing.jl (`] dev Turing`) in the build environment.

Reply:
The actual building workflow for the docs is described [here](https://github.com/TuringLang/turing.ml#building-the-documentation-site-locally) - it is likely that the `turing.ml` [repo](https://github.com/TuringLang/turing.ml/blob/master/make.jl) is using a slightly different setup from `Turing.jl`

Reply:
Yes I think the issue is in the differences in the markdown processing, which runs through `kramdown` in `Turing.jl`, while in `turing.ml` `CommonMarkGhPages` is used for markdown. That makes it obsolete to test `Turing.jl` with a local build from the repo itself. I'll try to run it through `turing.ml` and in the meantime update the instructions for local website build on `turing.ml` (slightly inaccurate with directory names) and `Turing.jl` (how to build it through `turing.ml`.

Reply:
According to https://github.com/TuringLang/turing.ml/issues/8#issue-824225056, here in Turing.jl we should just use Documenter-style syntax, ie. ``` `` ``` for inline math (no `$`) and ```` ```math ```` for display math (no `$$`). Of course, it would be nice to test it first though.

Reply:
Ok I figured out how to build the website through `turing.ml` locally and was able to finally test the latex display. I think it is finally fixed now. I will add a PR to `turing.ml` for the local build and a PR here to resolve the math issues.